### PR TITLE
Trip tips highlights fields

### PR DESCRIPTION
**Pull request by geoffreymm**

### CODE REVIEW

File: docs/reference/objects/product/trip_tip.md [hunk 1]
```diff
@@ -4,22 +4,20 @@ title: Trip Tip
...
 
 # trip_tip.body
 
-Returns the trip tips body
+Returns the trip tip's body.
 
 # trip_tip.heading
 
-Returns the trip tips heading
+Returns the trip tips heading.
```
Comment (tomchipchase) [inline docs/reference/objects/product/trip_tip.md:15]: Missing ' if we want to be consistent.

File: docs/reference/objects/product/trip_tip.md [hunk 2]
```diff
@@ -4,22 +4,20 @@ title: Trip Tip
...
 # trip_tip.id
 
-Returns a unique id for the trip tip
+Returns a unique id for the trip tip.
 
 # trip_tip.image
 
-Returns the trip tips [image]({% link docs/reference/objects/image.md %})
-
+Returns the trip tips [image]({% link docs/reference/objects/image.md %}).
```
Comment (tomchipchase) [inline docs/reference/objects/product/trip_tip.md:23]: Also here.

**Comment by geoffreymm** [comment]
[Update drops documentation on easol.dev](https://trello.com/c/glhZWKqz/53-update-drops-documentation-on-easoldev)

File: docs/reference/objects/product/highlight.md [hunk 3]
```diff
@@ -4,22 +4,20 @@ title: Highlight
...
 # highlight.id
 
-Returns a unique id for the highlight
+Returns a unique id for the highlight.
 
 # highlight.image
 
-Returns the highlights [image]({% link docs/reference/objects/image.md %})
-
+Returns the highlights [image]({% link docs/reference/objects/image.md %}).
```
Comment (tomchipchase) [inline docs/reference/objects/product/highlight.md:23]: Spotted another one, missed it last time.

**Review by tomchipchase — APPROVED** [review]

**Pull request merged**